### DISCUSSION TITLE
VST effect support

### DISCUSSION THREAD

**Discussion by Pomax** [File-New-Project/EarTrumpet#1002] [Ideas]
Any plans for supporting VST efffects, to do things like assigning a low pass filer and noise reduction to the audio out for your browser playing poorly mixed youtube videos, and a separate compressor on your FPS game audio out? It's the one thing that no one seems to bother with even though the use cases are so obvious, and would make life so much better given the incredible amount of high quality free VST out there. Even being able to just run OTT for the entire system would be a _huge_ gain. (pun intended, I guess)

**Comment by riverar** [comment]
Interesting idea, per-app VST. Thanks @Pomax! Will think about that some more.

**Comment by Pomax** [comment]
On the mac side of the fence, I'm using SoundSource for that purpose and while I have opinions about the app's UX, the fact that it's basically trivial to funnel any audio for any app through a VST chain makes it entirely worth the price. So that might be a second interesting thing: no VST? FOSS! Want VST, which is a Steinberg proprietary format? One time purchase fee, funding the dev effort.
